net-vpn/i2p/i2p-2.1.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="test"
+
+inherit java-pkg-2 java-ant-2 systemd
+
+DESCRIPTION="A privacy-centric, anonymous network"
+HOMEPAGE="https://geti2p.net"
+SRC_URI="https://files.i2p-projekt.de/${PV}/i2psource_${PV}.tar.bz2"
+
+LICENSE="Apache-2.0 Artistic BSD CC-BY-2.5 CC-BY-3.0 CC-BY-SA-3.0 EPL-1.0 GPL-2 GPL-3 LGPL-2.1 LGPL-3 MIT public-domain WTFPL-2"
+SLOT="0"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="nls"
+
+CP_DEPEND="
+	net-libs/nativebiginteger:0
```

Comment:
For [`getopt` and `json-simple`](https://github.com/i2p/i2p.i2p/blob/i2p-2.1.0/build.properties#L166-L171) in [`core/java/build.xml`](https://github.com/i2p/i2p.i2p/blob/i2p-2.1.0/core/java/build.xml#L36-L52) put
```
	dev-java/java-getopt:1
	dev-java/json-simple:0
```
For [`gnu/gettext/GettextResource.java`](https://github.com/i2p/i2p.i2p/blob/i2p-2.1.0/core/java/build.xml#L29-L35) there was [an attempt](https://github.com/gentoo/gentoo/pull/26163) which never got into the tree.

Reply:
> For [`gnu/gettext/GettextResource.java`](https://github.com/i2p/i2p.i2p/blob/i2p-2.1.0/core/java/build.xml#L29-L35) there was [an attempt](https://github.com/gentoo/gentoo/pull/26163) which never got into the tree.

hum, it seems that simply depending on gettext[java] works (± build issue on arm); did I miss smth?

> For [`jstl.jar` and `standard.jar`](https://github.com/i2p/i2p.i2p/blob/i2p-2.1.0/build.properties#L156-L159) in [`apps/susidns/src/lib/`](https://github.com/i2p/i2p.i2p/tree/i2p-2.1.0/apps/susidns/src/lib) feel free to use [this ebuild](https://github.com/gentoo/gentoo/files/10539773/jstl-1.2.7.ebuild.txt).

arf, I'm only proxy maint, I can't really add new ebuilds to the tree myself

Reply:
> > For [`gnu/gettext/GettextResource.java`](https://github.com/i2p/i2p.i2p/blob/i2p-2.1.0/core/java/build.xml#L29-L35) there was [an attempt](https://github.com/gentoo/gentoo/pull/26163) which never got into the tree.
> 
> hum, it seems that simply depending on gettext[java] works (± build issue on arm); did I miss smth?

Solving this issue might not be possible right now, just created https://bugs.gentoo.org/892698 as a reminder.

Reply:
> > For [`jstl.jar` and `standard.jar`](https://github.com/i2p/i2p.i2p/blob/i2p-2.1.0/build.properties#L156-L159) in [`apps/susidns/src/lib/`](https://github.com/i2p/i2p.i2p/tree/i2p-2.1.0/apps/susidns/src/lib) feel free to use [this ebuild](https://github.com/gentoo/gentoo/files/10539773/jstl-1.2.7.ebuild.txt).
> 
> arf, I'm only proxy maint, I can't really add new ebuilds to the tree myself

Don't regard it as a new package, it's a dependency needed for unbundling stuff from an existing package. And, adding it to this PR would [solve a bug](https://bugs.gentoo.org/892703).